pkg/cmd/roachtest/tests/failure_injection.go

```diff
@@ -614,6 +664,31 @@ func setupFailureSmokeTests(
 	if err := c.Install(ctx, t.L(), c.CRDBNodes(), "vmtouch"); err != nil {
 		return err
 	}
+
+	// Setup cgroup_exporter as a scrape target in prometheus.
+	// TODO(darryl): Once #143404 is complete, we should switch all cgroup
+	// performance assertions to use prometheus metrics instead of the current
+	// approach of sampling cgroup directly. There's nothing blocking us from
+	// using clusterstats to query for metrics, but the solution proposed in #143404
+	// is cleaner for this use case so lets not waste time on a temporary change.
+	promSetupLogger, _, err := roachtestutil.LoggerForCmd(t.L(), c.WorkloadNode(), "setup-prom")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	promCfg := &prometheus.Config{}
+	promCfg.WithPrometheusNode(c.WorkloadNode().InstallNodes()[0]).
+		WithCluster(c.CRDBNodes().InstallNodes()).
+		WithGrafanaDashboardJSON(dashboard.CgroupIOGrafanaJSON).
+		WithCgroupExporter(c.CRDBNodes().InstallNodes())
+
+	if err = c.StartGrafana(ctx, promSetupLogger, promCfg); err != nil {
```

Comment:
Do we really need to spin up internal prometheus and grafana? The change in `createLabels` adds the dynamic scrape target (via promhelper) to the centralized instance.